cmd/resolver-cli/cmd/resolve_test.go

```diff
@@ -0,0 +1,59 @@
+package cmd_test
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/paxthemax/resolver/cmd/resolver-cli/cmd"
+)
+
+func TestResolveCmd(t *testing.T) {
```

Comment:
I suppose that we can mock resolvers in this test to test only the code in this package, not also in others and to avoid resolving addresses by contacting external service.